packages/docs/src/styles/components/colorSwatchList.scss

```diff
@@ -12,3 +12,44 @@
   vertical-align: middle;
   width: $spacer-4;
 }
+
+.c-color-ramp__wrapper {
+  flex-shrink: 0;
+  margin-right: $grid-gutter-width;
+  width: 256px;
```

Comment:
**Alt solution**
If you applied `gap: $grid-gutter-width` to this element's parent `div`, you could get away with the following:
```css
.c-color-ramp__wrapper {
  flex-basis: 256px;
}
```

Reply:
Nice! Thanks